qiwi.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python3
 
 """
-Swift is a main manager for swift system.
+QIWI is a main manager for qiwi system.

```

Comment:
Maybe 'Swift' referred to the class, so `Qiwi` would be appropriate. ('qiwi system' is okay)